iroh-gossip/src/net.rs

```diff
@@ -469,13 +479,21 @@ impl Actor {
 
     async fn handle_in_event(&mut self, event: InEvent, now: Instant) -> anyhow::Result<()> {
         let me = *self.state.me();
-        debug!(me = ?me, "handle in_event  {event:?}");
+        if matches!(event, InEvent::TimerExpired(_)) {
+            trace!(me = ?me, "handle in_event  {event:?}");
```

Comment:
Those were not added in this PR but in #1149. `me` is a field because it is a simple type (32 byte peer id) whereas `event` is a complex type (enum with tuple variantes and potentially a lot of data in them) so I thought debug-printing makes more sense. But also I'm not really sure when to use fields vs debug logging in tracing. I created a separate PR for this as its unrelated to this PR: https://github.com/n0-computer/iroh/pull/1390